contrib/samba-aad-server/build/krb5.conf

```diff
@@ -0,0 +1,62 @@
+[libdefaults]                                                   
+  ticket_lifetime = 24h                                         
+  default_realm = FAREAST.CORP.MICROSOFT.COM                    
+  forwardable = true                                            
+                                                                
+[realms]                                                        
+  CORP.MICROSOFT.COM = {                                        
+    kdc = corp.microsoft.com                                    
+    default_domain = CORP.MICROSOFT.COM                         
+  }                                                             
+                                                                
+  EUROPE.CORP.MICROSOFT.COM = {                                 
+    kdc = europe.corp.microsoft.com                             
+    default_domain = CORP.MICROSOFT.COM                         
+  }                                                             
+                                                                
+  FAREAST.CORP.MICROSOFT.COM = {                                
+    kdc = fareast.corp.microsoft.com                            
```

Comment:
It seems that this is only a service could be deployed in the internal env, right?

Reply:
Yes, this is config is same as DLTS's winbind server